CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
-- Update codebase to use ES6
+-  Update dev package dependencies
```

Comment:
We use to include the detail of the update (i.e which dependency and from which verstion to which other, eg: "Upgrade logops dependency from 1.0.8 to 2.1.0").

However, in this case there is a lot of dependencies and maybe is a bit overwhelming. In that case, I suggest some general such as:

```suggestion
-  Overall update of dev package dependencies
```

Reply:
fixed c31f7f3